source-map-support.js

```diff
@@ -3,7 +3,11 @@ var path = require('path');
 
 var fs;
 try {
-  fs = require('fs');
+    fs = require('fs');
+    if (!fs.readFileSync) {
+        // If fs is empty, we are in the browser
```

Comment:
Well we don't use `browserify-fs` plugin so without this it we don't have a functionnal `fs`.
`node-browserify` default builtins are defined here https://github.com/substack/node-browserify/blob/8f5ebbf50544bdbdf34d4e0f336ac531152a5da4/lib/builtins.js

For the code comment I'll fix that right away